Rerender feedstock to matrix across Python versions

pkgw opened the pull request:
Not sure why it didn't do it the first time.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

pkgw committed:
Partially rerender feedstock to matrix across Python versions on Windows.

Not sure why it didn't do it before. We *don't* matrix on OSX and Linux
because the package does not actually depend on Python outside of its build
stage. But, on Windows we need to build variants with different versions of
Visual Studio.
